cohorts/cohort.py

```diff
@@ -500,29 +613,33 @@ def _load_single_patient_variants(self, patient, filter_fn):
                     variant_collections.append(patient_variants)
                 else:
                     raise ValueError("Don't know how to read %s" % patient_variants)
+            # merge variant-collections
+            if len(variant_collections) == 0:
+                failed_io = True
```

Comment:
Since this isn't actually failed IO, rename the variable to `no_variants` or something?